### ISSUE TITLE
Deleting multiple files is sometimes slow on macOS

### ISSUE THREAD

**Issue by matangover** [fman-community/fman#587]
When I select two files and press Cmd+Delete, there is a noticeable lag of 5-10 seconds before the files are deleted. During the lag, sometimes the spinning cursor appears and sometimes not. fman's "file operation" dialog box is not shown.

This doesn't happen all the time, but if it happens once, there's a good chance it will happen again shortly after. I can reproduce it quite reliably.

Also, I'm not sure but I think that during the lag, if I press any key, the lag is cut and the files are immediately deleted.

I think the files themselves don't matter. The deleted files are small (less than 1 MB), and I've tried with the same files multiple times, sometimes it happens and sometimes not. It happens in normal folders containing a small amount of files.

Running macOS 10.14.1.

**Comment by mherrmann** [comment]
Is anybody else experiencing this? (Please chime in if you do.)

**Comment by fdb** [comment]
Yes! Here's a video showing that deleting files takes about 1-2 seconds *per file*:

https://enigmeta.s3.amazonaws.com/github-issues/2019-10-21-fman-slow-deletion.mp4

To compare, in the Finder it's about 1 second to delete about 1500 files. In the Terminal it's near instant.
